Resources/Prototypes/Damage/modifier_sets.yml

```
@@ -349,3 +349,14 @@
     Holy: 1.5
   flatReductions:
     Cold: 3
+
+- type: damageModifierSet
+  id: FireSpirit
+  coefficients:
+    Heat: 0.0
+    Cold: 0.5
+    Shock: 0.0
+    Blunt: 0.5
+    Slash: 0.5
+    Piercing: 0.5
+    Holy: 3.0
```

```suggestion
  coefficients:
    Heat: 0
    Cold: 1.2
    Blunt: 0.6
    Slash: 0.6
    Piercing: 0.6
    Holy: 1.5
```
Technically, cold doesn't currently exist in any meaningful way(Ifrits are immune to the cold of space), but these Values put the ifrit at 200 effective HP for damage types it is resistant to. They shouldn't be immune to shock damage though. If anything the various "Magic" damage types should do full damage to it.

Holy needs to be toned back to 1.5, because at 3x, anti-psychic weapons will pretty much instantly kill it.

---

My thought for the shock stuff was that Ifrit is made up of floating rocks, electricity wouldn't really hurt those